### DISCUSSION TITLE
Can the profile-level-id in SDP Negotiation be changed?

### DISCUSSION THREAD

**Discussion by LanTenDragon** [obsproject/obs-studio#9230] [Development Questions] [answered]
Currently for OBS streaming to WHIP endpoint, OBS only offers profile-level-id from constrained baseline `42e01f`

`a=fmtp:96 profile-level-id=42e01f;packetization-mode=1;level-asymmetry-allowed=1`

Even if the profile level has been set in the `Settings -> Output -> Streaming -> Profile` drop down menu
OBS still offers constrained baseline only
![Untitled](https://github.com/obsproject/obs-studio/assets/43511642/51e8c0ad-5a24-4320-9665-625bdf9f7df9)

Is this behaviour intended? Can it be changed?

**Comment by pkviet** [comment]
@Sean-Der @tt2468 are the main authors of whip output. Pinging them: could you offer some opinion ?

**Comment by Sean-Der** [comment, marked as answer]
Sorry about that @LanTenDragon!

Yes we should fix that. If we know the profile that is being used we should communicate that properly.

I would like to wait for User-Agent change by C++ migration. What do you think @pkviet ?

**Reply by pkviet** [reply]
Yeah we can wait for the c++ api migration.

**Comment by dmakeev** [comment]
Hello, @Sean-Der. Any update on it?
